internal/pkg/template/workload.go

```diff
@@ -483,6 +489,114 @@ func (cfg *ALBListener) RulePaths() []string {
 	return rulePaths
 }
 
+// ConditionGroups returns a slice of all the Splits on the Conditions across multiple listener rules.
+func (cfg *ALBListener) ConditionGroups(wkldType string, isHTTPS bool) []string {
```

Comment:
Based on what it does, I would probably name this function something like `GroupCountPerRule`. `ConditionGroups` implies that the return value is a list of condition groups, not a list of ints.